pkg/client/clientset/versioned/clientset.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
2023

Reply:
We must update https://github.com/kubernetes-sigs/kueue/blob/main/hack/boilerplate.go.txt if we want to use `2023` in this code.

This means we update the copyright of all already generated files.

So, I'm on the fence about whether we should update https://github.com/kubernetes-sigs/kueue/blob/main/hack/boilerplate.go.txt.

@alculquicondor wdyt?

Reply:
Uhm... that's a good point. In that case older seems better

Reply:
agree.